README.md

```diff
@@ -17,6 +17,8 @@ OTEAPI Core provides the core functionality of OTEAPI, which stands for the *Ope
 
 It uses the [strategy](https://en.wikipedia.org/wiki/Strategy_pattern) software design pattern to implement a simple and easy to extend access to a large range of data resources.
 Semantic interoperability is supported via mapping of data models describing the data to ontologies.
+A set of strategy interfaces that can be considered abstract classes for the implementation of strategies,  and data models used in their configuration, are provided.
```

Comment:
I am not sure this is so elightening. oteapi-core is actually not using abstract base classes, but rather protocol classes.